beacon-chain/blockchain/receive_block.go

```diff
@@ -141,7 +142,7 @@ func (s *Service) ReceiveBlockNoPubsubForkchoice(ctx context.Context, block *eth
 		return errors.Wrap(err, "could not get signing root on received block")
 	}
 
-	if !bytes.Equal(root[:], s.HeadRoot()) {
+	if featureconfig.Get().InitSyncCacheState && !bytes.Equal(root[:], s.HeadRoot()) {
```

Comment:
Should this be `!featureconfig.Get().InitSyncCacheState`?

Reply:
Oh I removed it. It's out of scope of the current PR : )